assignments/assignment-11/ifElse.js

```diff
@@ -0,0 +1,38 @@
+// 1. Password Correctness Check
+var correct = "abdullajon123";
+var correct2 = "Abdullajon123";
```

Comment:
Change variable name correct2 to something else. because it is not correct, but user input